hicexplorer/lib/utilities.py

```diff
@@ -0,0 +1,38 @@
+import logging
+log = logging.getLogger(__name__)
+
+
+class Utilities():
```

Comment:
Why should this be a class?

Reply:
I prefer to use an object oriented programming model, it makes the code better structured and more readable.